roles/lib_openshift/library/oc_project.py

```diff
@@ -1543,19 +1543,20 @@ def needs_update(self):
     def run_ansible(params, check_mode):
         '''run the idempotent ansible code'''
 
-        _ns = None
-        if params['node_selector'] is not None:
-            _ns = ','.join(params['node_selector'])
-
-        pconfig = ProjectConfig(params['name'],
-                                'None',
-                                params['kubeconfig'],
-                                {'admin': {'value': params['admin'], 'include': True},
-                                 'admin_role': {'value': params['admin_role'], 'include': True},
-                                 'description': {'value': params['description'], 'include': True},
-                                 'display_name': {'value': params['display_name'], 'include': True},
-                                 'node_selector': {'value': _ns, 'include': True},
-                                })
+        node_selector = ','.join(params.get('node_selector', [])) or None
+
+        pconfig = ProjectConfig(
+            params['name'],
+            'None',
+            params['kubeconfig'],
+            {
+                'admin': {'value': params['admin'], 'include': True},
+                'admin_role': {'value': params['admin_role'], 'include': True},
+                'description': {'value': params['description'], 'include': True},
+                'display_name': {'value': params['display_name'], 'include': True},
+                'node_selector': {'value': node_selector, 'include': True},
+            },
+        )
```

Comment:
Maybe one day we'll have something like https://github.com/google/yapf (or any alternative) in place and stop talking about formatting...

I'm not actively looking for code and reformatting, but since I touched this file and simplified the expression above, there it goes.

I explained the reasoning in another PR I was reviewing recently: https://github.com/openshift/openshift-ansible/pull/3461#discussion_r106629152
Or, in case the comment link stops working, http://blog.rodolfocarvalho.net/2017/03/indentation-styles.html.